doxia-integration-tools/src/main/java/org/apache/maven/doxia/tools/SiteTool.java

```diff
@@ -63,8 +64,11 @@ Artifact getSkinArtifactFromRepository( ArtifactRepository localRepository,
      * Get a site descriptor from the project's site directory.
      *
      * @param siteDirectory the site directory, not null
-     * @param locale the locale wanted for the site descriptor. If not null, searching for
-     * <code>site_<i>localeLanguage</i>.xml</code>, otherwise searching for <code>site.xml</code>.
+     * @param locale the locale wanted for the site descriptor, not null. Most specific
```

Comment:
the "most specific to least specific" part should be associated to the `@return` description